recipes/xorg-libximerna/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.1.4" %}
+
+package:
+  name: xorg-libximerna
```

Comment:
This package name is mistaken! It is "Xinerama" (like Cinerama), not "Ximerna". This appears in several places.

Reply:
yeah. I clearly can't read.